### PR TITLE
Add a CITATION.cff

### PR DESCRIPTION
**Pull request by DonggeLiu**
Similar to https://github.com/google/oss-fuzz/pull/11851,
Based on https://dl.acm.org/doi/10.1145/3468264.3473932.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    affiliation: Google LLC
+    orcid: 'https://orcid.org/0000-0002-7042-0444'
+  - given-names: Laszlo
+    family-names: Szekeres
+    email: lszekeres@google.com
+    affiliation: Google LLC
+    orcid: 'https://orcid.org/0000-0002-9808-6484'
+  - given-names: Laurent
+    family-names: Simon
+    email: laurentsimon@google.com
```
Comment (DonggeLiu) [inline CITATION.cff:23]: Is there a preferred alternative email?
Or is it preferred to omit emails from this repo?

Comment (oliverchang) [reply CITATION.cff:20]: Probably just omit.

**Review by oliverchang — APPROVED** [review]

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]
I'm not so sure we need it here. There's a fuzzbench paper people can cite.

**Comment by DonggeLiu** [comment]
I see, the paper's DOI in the file.
I will close this for now, then : )

**Pull request closed without merging**